packages/cli/src/lib/build-strategies/strategies/DockerVMStrategy.ts

```diff
@@ -17,7 +17,9 @@ import fse from "fs-extra";
 import path from "path";
 import Mustache from "mustache";
 
-type BuildableLanguage = Exclude<PolywrapManifestLanguage, "interface">;
+type BuildableLanguage =
```

Comment:
This is a redefinition of the type defined in packages/cli/src/lib/project/manifests/polywrap/languages.ts